src/expandableCalendar/__test__/expandableCalendarTestKit.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import {
+  CalendarContextProviderProps,
+  CalendarProvider,
+  ExpandableCalendar,
+  ExpandableCalendarProps
+} from 'react-native-calendars';
+import {toMarkingFormat} from '../../interface';
+
+const XDate = require('xdate');
+
+const today = new XDate();
+export const testIdExpandableCalendar = 'myExpandableCalendar';
+
+export const expandableCalendarTestIDs = (testId: string) => {
```

Comment:
Can't we move these to the driver? Why do we need another file?

Reply:
I believe it's easier to read when the driver itself (accessing components and performing actions on them) should be separated from the test kit (where we create all the mocks and the component we want to render in the driver)